cli/printers/base_printer.py

```diff
@@ -12,5 +14,5 @@ def __init__(self, context: click.Context):
         self.context = context
 
     @abstractmethod
-    def print_results(self, results: List[DocumentDetections]):
+    def print_results(self, context: click.Context, results: List[DocumentDetections]):
```

Comment:
the code changed. please merge from dev!

Reply:
Fixed